.github/workflows/release.yml

```diff
@@ -42,7 +46,29 @@ jobs:
           GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GRADLE_ENTERPRISE_CACHE_PASSWORD }}
         run: |
           echo $GPG_FILE | base64 -d > secring.gpg
-          ./gradlew publishToSonatype closeAndReleaseSonatypeStagingRepository
+          # Publish both locally and to Sonatype.
+          # The artifacts stored locally will be used to generate the SLSA provenance.
+          ./gradlew publishToMavenLocal publishToSonatype closeAndReleaseSonatypeStagingRepository
+          # Read the current version from gradle.properties.
+          VERSION=$(./gradlew properties | grep 'version:' | awk '{print $2}')
```

Comment:
I may be wrong, but I think that if you use the `publishToBuildRepository` task, then you don't need to search for the group or version anymore, and you can use the `build/repo` directory as the upload source for GH.

Reply:
I still need to read the group because I need to resolve the path to the artifacts for uploading them as artifacts. The path can be different in different repositories that use this template.

I read the version just to make sure the artifacts are collected from the right path, but that is just to be extra cautious.